css/footer.css

```diff
@@ -0,0 +1,140 @@
+footer {
+  width: 100%;
+  background: var(--black);
+  margin-top: 2em;
+  font-size: 5vw;
+  font-weight: bolder;
+  color: var(--white);
+}
+.footer h1 {
+  letter-spacing: 0.3em;
+  font-weight: 100;
+}
+.footer_info {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+.footer_item {
+  padding-top: 3em;
+}
+.footer_item ul {
+  padding-top: 1em;
+}
+.footer_item ul li {
+  list-style: none;
+  margin-top: 5px;
+}
+.footer_item ul li a {
+  color: white;
+}
+.footer_item h1 {
+  font-size: 8vw;
+}
+.footer_form {
+  margin-top: 3em;
+  padding: 0 1em;
+  display: flex;
+  flex-direction: column;
+}
+
+.footer_item h1 {
+  font-size: 5.5vw;
+}
+.footer_form form {
+  width: 100%;
+  margin-top: 1em;
+  align-self: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.footer_form input {
+  background: transparent;
+  border: 1px solid var(--white);
+  padding: 2em 0;
+  margin: 0.5em 0 1em;
+  font-size: 2vw;
+}
+
+.footer_form h1 {
+  text-align: center;
+}
+.footer_form label {
+  display: block;
+}
+.form_submit {
+  background: var(--white);
+  color: var(--black);
+  padding: 1em;
+  cursor: pointer;
+}
+.social_icons {
+  width: 100%;
+  padding: 4em;
+}
+.social_icons ul {
+  display: flex;
+  text-align: center;
+  width: 80%;
+  margin: 0 auto;
+}
+.social_icons ul li {
+  list-style: none;
+}
+.social_icons ul li a {
+  display: block;
+}
+.social_icons ul li a img {
+  width: 50%;
+  max-width: 100%;
+}
```

Comment:
It's generally considered best practice to avoid too much nested selectors. If you want to target this kind of element, it might be best just to give the `img` its own class, and target that directly.